common/primitives/src/schema.rs

```diff
@@ -48,6 +67,8 @@ pub struct SchemaResponse {
 	pub model_type: ModelType,
 	/// The payload location
 	pub payload_location: PayloadLocation,
+	/// grants for the schema
+	pub settings: Vec<SchemaSetting>,
```

Comment:
@aramikm  i am a bit divided here, to support SchemaSetting directly we would definitely need a custom serializer and de for Bitflags<SchemaSetting, u16> given that what essentially SchemaSettings is

I was under the impression to return actual types as vec to prevent doing all above 

What you suggest, or i can set a ticket out to fix this?

Reply:
Another thing is in our users mind they are setting some values and exactly what they get back I think that will keep consistency than serializing flags what you think?

Reply:
Looks like this never translates back I get nil so this needs fix before merge 

Will do that

Reply:
I think this is fine. It might be confusing if the input and output is not consistent from user perspective as you pointed out.